package.json

```diff
@@ -162,6 +162,8 @@
     "@tsconfig/node12": "^1.0.7",
     "@tsconfig/node14": "^1.0.0",
     "@tsconfig/node16": "^1.0.1",
+    "acorn": "^8.4.1",
+    "acorn-walk": "^8.1.1",
```

Comment:
How much does this affect install size?

Reply:
Now sure how you calculate "install size", but here's the node_modules before and after.

1.9M    node_modules
2.5M    node_modules

Reply:
Ok, that looks correct based on the sizes of https://www.npmjs.com/package/acorn and https://www.npmjs.com/package/acorn-walk

I was going to say we should avoid adding dependencies, and then I remembered that TypeScript is 60MB.  So, uhhhh... nevermind :)